qlib/data/ops.py

```diff
@@ -1384,6 +1385,8 @@ def exp_weighted_mean(x):
             series = series.expanding(min_periods=1).apply(exp_weighted_mean, raw=True)
         elif 0 < self.N < 1:
             series = series.ewm(alpha=self.N, min_periods=1).mean()
+        elif self.alpha and 0 < self.alpha < 1:
+            series = series.ewm(alpha=self.alpha, min_periods=1).mean()
```

Comment:
What extra feature does it provide? 
Does the original version provide the same feature if `0 < self.N < 1`?

Reply:
Yes, you are right. I misunderstand what the meaning when the 0< self.N < 1.
I can use the EMA with 1/N when N > 1.
Close the PRs.
Thanks for your feedback. :)